builtin/logical/database/path_config_connection.go

```diff
@@ -392,6 +393,7 @@ func (b *databaseBackend) connectionReadHandler() framework.OperationFunc {
 		}
 
 		resp.Data = structs.New(config).Map()
+		delete(resp.Data, "verify_connection")
```

Comment:
This seems like something we could keep in the response.

Reply:
I was on the fence on this as well. I agree we can probably keep it so I'll remove that line!